jobfunnel/config/parser.py

```diff
@@ -47,6 +47,25 @@ def parse_cli():
                         required=False,
                         help='list of keywords to use in the job search. ('
                              'i.e. Engineer, AI)')
+    ''' adding additional cli options '''
+    parser.add_argument('-p',
+                        dest='province',
+                        type=str,
+                        required=False,
+                        help='Province value for a region ')
+
+    parser.add_argument('-city',
```

Comment:
For long command options like `-city` it is convention to use `--city` instead.  Unfortunately `-c` is already taken. 😢